result/src/main/kotlin/com/github/kittinunf/result/Result.kt

```diff
@@ -64,7 +91,17 @@ inline fun <V : Any?, E : Exception> Result<V, E>.onError(f: (E) -> Unit) = when
     }
 }
 
-fun <V : Any?, E : Exception> Result<V, E>.any(predicate: (V) -> Boolean): Boolean = try {
+inline fun <V : Any?, E : Exception> Result<V, E>.onSuccess(f: (V) -> Unit): Result<V, E> {
```

Comment:
Do we need to have `onFailure` for the sake of completeness?

Reply:
Yep, there exists already an `onError`.

Reply:
Oh sorry I think I missed that 👍